spec/game_spec.rb

```diff
@@ -0,0 +1,42 @@
+require_relative '../classes/game'
+
+RSpec.describe Game do
+  let(:publish_date) { Date.new(2022, 1, 1) }
+  let(:last_played_at) { DateTime.new(2023, 1, 1) }
+  let(:game) { Game.new(true, last_played_at, publish_date) }
+
+  describe '#initialize' do
+    it 'creates a new game with specified attributes' do
+      expect(game.multiplayer).to eq(true)
+      expect(game.last_played_at).to eq(last_played_at)
+      expect(game.publish_date).to eq(publish_date)
```

Comment:
You will also have to convert the dates back to strings like this:

```
expect(game.last_played_at.to_s).to eq(last_played_at)
expect(game.publish_date.to_s).to eq(publish_date)
```

This is needed because our input String is converted to a Date object in the `Game` class, so it must be converted back to String to compare it with the input.